hack/quick-install.sh

```diff
@@ -15,35 +15,28 @@ kubectl apply -f https://github.com/knative/eventing/releases/download/knative-$
 kubectl rollout status deployment --namespace=cert-manager --timeout=30s
 kubectl rollout status deployment --namespace=knative-eventing --timeout=30s
 
-kubectl apply -f - << EOF
-apiVersion: v1
-kind: ConfigMap
-metadata:
-  name: config-features
-  namespace: knative-eventing
-  labels:
-    eventing.knative.dev/release: devel
-    knative.dev/config-propagation: original
-    knative.dev/config-category: eventing
-data:
-  new-apiserversource-filters: enabled
-EOF
-
 bash cmd/operator/generate.sh
 helm template kubearchive charts/kubearchive -n kubearchive \
     --include-crds \
     --set "global.production=true" > /tmp/kubearchive-not-resolved.yaml
 ko resolve -f /tmp/kubearchive-not-resolved.yaml --base-import-paths > /tmp/kubearchive.yaml
 kubectl apply -n kubearchive -f /tmp/kubearchive.yaml
 
-cat << EOF > /tmp/patch.yaml
-stringData:
-  POSTGRES_PORT: "5432"
-  POSTGRES_URL: postgresql.databases.svc.cluster.local
-  POSTGRES_USER: kubearchive
-  POSTGRES_DB: kubearchive
-  POSTGRES_PASSWORD: 'P0stgr3sdbP@ssword'  # notsecret
-EOF
-kubectl patch -n kubearchive secrets kubearchive-database-credentials --patch-file /tmp/patch.yaml
-kubectl rollout restart deployment --namespace=kubearchive kubearchive-sink kubearchive-api-server
+kubectl -n kubearchive rollout status deployment --timeout=90s
+kubectl -n kubearchive wait pod --all --for=condition=ready --timeout=90s
+
+# Make sure webhooks are up and running.
+LOCAL_PORT=8443
+kubectl -n kubearchive port-forward service/kubearchive-operator-webhooks ${LOCAL_PORT}:443 >& /dev/null &
+
+# Wait for $LOCAL_PORT to become available.
+while ! nc -vz localhost ${LOCAL_PORT} > /dev/null 2>&1 ; do
+    echo -n .
+    sleep 0.5
+done
+echo .
```

Comment:
This checks made me think that we probably need to better configure the `/readyz` endpoint of the operator to do that, so the rollout and the wait made this work for us as the deployment "should know" when it is ready to provide service.

IINM, we have currently a [dummy function](https://github.com/maruiz93/kubearchive/blob/main/cmd/operator/main.go#L147) `healthz.Ping` that always returns true, but I guess we could use a more sophisticated check if we need.

Of course this is out of the scope of the PR, but just checking if we need to create a new issue to address this.

Reply:
@maruiz93 , create an issue for this for the operator, and I'll take a look.  I only added this because I know that the webhooks have to be responding before we can apply a `KubeArchveConfig` and have things work.  If it's not things don't work.

But other components may have similar endpoints that need this check too.  Or the probe.

Reply:
Done https://github.com/kubearchive/kubearchive/issues/439